x-pack/plugins/ml/public/application/services/results_service/result_service_rx.ts

```diff
@@ -263,57 +262,59 @@ export function resultsServiceRxProvider(mlApiServices: MlApiServices) {
       ];
 
       return mlApiServices.results
-        .anomalySearch$({
-          index: ML_RESULTS_INDEX_PATTERN,
```

Comment:
Might be missing some context but I'm not quite understanding why index is not needed here anymore?

Reply:
Ah, I see it's been moved into `anomalySearch`.  Disregard the question.